ui_testing/pages/header_page.py

```diff
@@ -0,0 +1,280 @@
+            self.base_selenium.LOGGER.info('Press on the header button')
+            self.base_selenium.click(element='header:header_button')
+            self.sleep_small()
+
+    def click_on_user_management_button(self):
+            self.base_selenium.LOGGER.info('Press on the user management button')
+            self.base_selenium.click(element='header:user_management_button')
+            self.sleep_small()
+
+    def archive_selected_users(self):
+            self.base_selenium.scroll()
+            self.base_selenium.click(element='user_management:right_menu')
+            self.base_selenium.click(element='user_management:archive')
+            self.confirm_popup()
+
+    def get_archived_users(self):
+            self.base_selenium.scroll()
+            self.base_selenium.click(element='user_management:right_menu')
+            self.base_selenium.click(element='user_management:archived')
+            self.sleep_small()
+
+    def is_user_in_table(self, value):
+            """
+                - get_archived_test_units then call me to check if the test unit has been archived.
+                - get_active_test_units then call me to check if the test unit is active.
+            :param value: search value
+            :return:
+            """
+            results = self.search(value=value)
+            if len(results) == 0:
+                return False
+            else:
+                if value in results[0].text:
+                    return True
+                else:
+                    return False
+
+    def restore_selected_user(self):
+            self.base_selenium.scroll()
+            self.base_selenium.click(element='user_management:right_menu')
+            self.base_selenium.click(element='user_management:restore')
+            self.confirm_popup()
+
+    def get_active_users(self):
+            self.base_selenium.scroll()
+            self.base_selenium.click(element='user_management:right_menu')
+            self.base_selenium.click(element='user_management:active')
+            self.sleep_small()
+
+
+    def create_new_user(self, user_role='', sleep=True, user_email='', user_password='', user_confirm_password='', user_name=''):
+        self.base_selenium.LOGGER.info(' + Create new user.')
+        self.base_selenium.click(element='user_management:create_user_button')
+        time.sleep(self.base_selenium.TIME_SMALL)
+        self.user_name = self.generate_random_text()
+        self.set_user_name(self.user_name)
+        self.set_user_email(user_email)
+        self.set_user_role(user_role)
+        self.set_user_password(user_password)
+        self.set_user_confirm_password(user_confirm_password)
+
+        user_data = {
+            "user_name":self.get_user_name(),
+            "user_email":self.set_user_email,
+            "user_role": self.get_user_role(),
+            "user_password": self.get_user_password(),
+            "user_confirm_password": self.get_user_confirm_password()
+        }
```

Comment:
use the values which u used in `set` methods. Actually the set methods return this value